ran-core/src/main/java/io/ran/MappingClassWriter.java

```diff
@@ -92,7 +98,8 @@ private void createCopy() {
 				w.invoke(getter);
 				w.invoke(setter);
 			}
-			w.returnNothing();;
+			w.returnNothing();
+			;
```

Comment:
Extra semi-colon, remove

Reply:
yeah, i wanted to keep it just to white space, and other types do later, but I can add this one